src/builtins_system.rs

```diff
@@ -196,6 +197,22 @@ fn builtin_jobs(
     }
 }
 
+fn builtin_sys_time(
```

Comment:
This should be 'builtin_epoch' following a consistent pattern between the sl-sh name and builtin name simplifies finding builtins later when you forget where they are.